cli/test/lib/tasks/verify_spec.js

```diff
@@ -779,6 +781,7 @@ function createfs ({ alreadyVerified, executable, packageVersion, customDir }) {
         },
       },
     },
+    [nodeModulesFolder]: mockfs.load(nodeModulesFolder),
```

Comment:
listr dynamically requires modules, this broke in Node 14, this change excludes node_modules from mock-fs